client/lib/react-helpers/index.js

```diff
@@ -35,7 +36,11 @@ export function renderWithReduxStore( reactElement, domContainer, reduxStore ) {
 			: domContainer;
 
 	return ReactDom.render(
-		React.createElement( ReduxProvider, { store: reduxStore }, reactElement ),
+		<ReduxProvider store={ reduxStore }>
+			<Stylizer onInsertCss={ insertCss }>
```

Comment:
I would move the `Stylizer` to `renderPage` in principle so `renderWithReduxStore` does what it says it does. But I think we don't use `renderPage` everywhere yet so that's for another PR.